sdk/helper/testcluster/util.go

```diff
@@ -0,0 +1,335 @@
+// Copyright (c) HashiCorp, Inc.
+// SPDX-License-Identifier: MPL-2.0
+
+package testcluster
+
+import (
+	"context"
+	"encoding/base64"
+	"encoding/hex"
+	"fmt"
+	"sync/atomic"
+	"time"
+
+	"github.com/hashicorp/go-uuid"
+	"github.com/hashicorp/vault/api"
+	"github.com/hashicorp/vault/sdk/helper/xor"
+)
+
+// Note that OSS standbys will not accept seal requests.  And ent perf standbys
+// may fail it as well if they haven't yet been able to get "elected" as perf standbys.
+func SealNode(ctx context.Context, cluster VaultCluster, nodeIdx int) error {
+	if nodeIdx >= len(cluster.Nodes()) {
+		return fmt.Errorf("invalid nodeIdx %d for cluster", nodeIdx)
+	}
+	node := cluster.Nodes()[nodeIdx]
+	client := node.APIClient()
```

Comment:
I notice this has the same issues I've fixed in #20048 -- I was thinking, perhaps its best to use `client.WithNamepace("")` here to consistently access `sys/`? 

Its a little more work for the util author here, but means potentially less thought for test writers who just call `SetNamespace` without thinking about it.

Reply:
What do you think about APIClient() always cloning the stored client instead?

Reply:
I'm curious if this breaks anything in general, but I like the idea :-)

Reply:
I remembered why I didn't do this: it would require error handling.  Panic to the rescue!